packages/trpc/server/routers/viewer.tsx

```diff
@@ -587,6 +587,7 @@ const loggedInViewerRouter = createProtectedRouter()
           status: {
             notIn: [BookingStatus.CANCELLED],
           },
+          userId: user.id,
```

Comment:
Taking the chance to fix this, it was a bug.

Reply:
awesome thank you!